.env.example

```
@@ -64,6 +64,9 @@ NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
 POSTHOG_DEBUG=false
 POSTHOG_SESSION_RECORDING_ENABLED=true
 POSTHOG_AUTO_CAPTURE=false
+# PostHog API (for server-side analytics data fetching in admin dashboard)
+POSTHOG_PERSONAL_API_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTHOG_PERSONAL_API_KEY key should go before the POSTHOG_SESSION_RECORDING_ENABLED key

(UnorderedKey)